### DISCUSSION TITLE
Official chat platform?

### DISCUSSION THREAD

**Discussion by rdrpenguin04** [slint-ui/slint#1044] [Ideas] [answered]
It would be nice to have a chat platform (such as Discord, Slack, or Zulip, or even multiple) that would allow people working on projects using this library to communicate and ask each other questions. I would be willing to set up a Discord server, though I imagine other people would be more qualified and/or have more time to moderate.

**Comment by tronical** [comment, marked as answer]
Hi @rdrpenguin04 ! We’re hosting a mattermost chat instance at https://chat.slint-ui.com/ . Feel free to join us there :)

**Reply by qewer33** [reply]
I don't know the team's view on Discord but it would be great to have an official Discord server for Slint as well. It would make chatting more accessible since there are more people that use Discord (+ there are a few popular Rust Discord servers so the majority of the Rust community that is interested in Slint already use Discord)

**Reply by hunger** [reply]
Discord is nice enough, but it is also closed source. We will loose some existing users if we switched there.

I really do not want that to happen, and I do not want to split the community over two services either.
